tests/phpunit/testdata/plugins/test-plugin-plugin-readme-errors-invalid-name/readme.txt

```diff
@@ -0,0 +1,12 @@
+=== Plugin Name ===
```

Comment:
And why is this invalid? 🤔

Reply:
https://github.com/WordPress/wordpress.org/blob/trunk/wordpress.org/public_html/wp-content/plugins/plugin-directory/readme/class-parser.php#L227 `if ( 'plugin name' == strtolower( $this->name ) ) {
			$this->warnings['invalid_plugin_name_header'] = true; ...`

Reply:
Ah 💡